comb_spec_searcher/proof_tree.py

```diff
@@ -712,6 +712,51 @@ def _recursion_fixer(self, css, root=None, in_labels=None):
         for child in root.children:
             self._recursion_fixer(css, child, in_labels)
 
+    def expand_tree(self, pack, function_kwargs=dict(), **kwargs):
+        """
+        Return a ProofTree that comes from expanding the strategy verified
+        combinatorial classes using the StrategyPack 'pack'.
+
+        The function relies on CombinatorialSpecificationSearcher and its
+        method 'auto_search'. It first created a universe with rules from
+        'self', and then adds strategy_verified objects to the front of the
+        queue. All classes are assumed to be expandable. The '**kwargs' are
+        passed to the 'auto_search' method.
+        """
+        from .comb_spec_searcher import CombinatorialSpecificationSearcher
+        root_class = self.root.eqv_path_comb_classes[0]
+        css = CombinatorialSpecificationSearcher(
```

Comment:
What do you think of extracting the searcher from the `kwargs` and defaulting to `CombinatorialSpecificationSearcher` if none is given. It would allow to run the expansion with `TileScope` when we have a `TileScopePack`

Reply:
This makes sense. Are we ok with importing in the function? It is the easy way around the circular dependency.

Reply:
I agree that it's the best way to avoid circular dependency